ui/components/Modal.js

```diff
@@ -0,0 +1,41 @@
+import { Modal, Backdrop, Fade } from '@mui/core'
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+
+const CustomModalWrapper = styled(Modal)(({ theme }) => ({
+   display : "flex",
+   alignItems : "center",
+   justifyContent : "center", 
+}))
+
+/**
+ *
+ * @param {{
+ *  open?: boolean,
+ *  Content?: JSX.Element,
+ *  handleClose?: (event: {}, reason: "backdropClick" | "escapeKeyDown") => void,
+ *  container?: React.ReactInstance | (() => React.ReactInstance)
+ * }} props
+ * @returns 
+ */
+
+export const CustomModal = ({
+    open, Content, handleClose, container
+  }) =>  { const theme = useTheme();
+    return (
+      <CustomModalWrapper
+        sx={{theme}}
```

Comment:
what's the purpose of this ?

Reply:
This styled api thing is confusing the hell out of me, I thought this is how the style is called🤦, but now that i use my small little brain I understand that <CustomModalWrapper /> tag would do that, right?

Reply:
yes, you actually need not use that `sx` prop here

Reply:
other than that, everything else is fine...

Reply:
Okay this is good?